graph/src/data/schema.rs

```diff
@@ -190,3 +979,210 @@ fn invalid_interface_implementation() {
          required fields: x: Int, y: Int"
     );
 }
+
```

Comment:
Was this file formatted with `rustfmt`? I'm spotting few suspiciously long lines, but I could be wrong.

Reply:
It is to spec!